tests/test_openjdk.py

```diff
@@ -28,3 +30,39 @@ def test_jdk_version(container, java_version):
     assert (
         container.connection.check_output("echo $JAVA_VERSION") == java_version
     )
+
+@pytest.mark.parametrize(
+    "container_per_test",
+    [
+        OPENJDK_11_CONTAINER,
+        OPENJDK_17_CONTAINER,
+    ],
+    indirect=["container_per_test"]
+)
+@pytest.mark.parametrize(
+    "test_to_run, expected_string",
+    [
+        ("threads_concurrency_and_sleep" , ["I am the thread 1", "I am the thread 2"])
+    ],
+)
+def test_jdk_extended(
```

Comment:
Please add some documentation for this test.

Reply:
Done

Reply:
done